app/src/main/java/com/yuneec/example/component/actitivty/MainActivity.java

```diff
@@ -128,7 +128,7 @@ public void publishBatteryChangeStatus(final String batteryStatus) {
             @Override
             public void run() {
 
-                Log.d(TAG, batteryStatus);
+                //Log.d(TAG, batteryStatus);
```

Comment:
I am always in favor of removing instead of commenting out a line :-). Especially for logging. There is also `Log.v`.

Reply:
You're right. Will go through all these some time.